rotkehlchen/tests/pylint/test_lognokwargs.py

```diff
@@ -19,7 +19,7 @@ def test_simple_logger_with_kwargs(pylint_test_linter):
 
         checker.visit_call(node)
         messages = checker.linter.release_messages()
-        assert len(messages) == 1
+        assert len(messages) == 2
```

Comment:
This is probably a bug in either pylint or in our module. That's not theright solution ;)